server/router.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
```

Comment:
that is good to have a router, but, i think is a better idea to create different files to keep only routes for categories and same with all other models, what do you think? let me know if that sounds good to you too